Packs/ApiModules/Scripts/TAXII2ApiModule/TAXII2ApiModule.py

```diff
@@ -1219,20 +1226,34 @@ def load_stix_objects_from_envelope(self, envelopes: types.GeneratorType, limit:
         if relationships_lst:
             indicators.extend(self.parse_relationships(relationships_lst))
         demisto.debug(
-            f"TAXII 2 Feed has extracted {len(indicators)} indicators"
+            f"TAXII 2 Feed has extracted {len(list(indicators))} indicators"
         )
 
         return indicators
 
+    def increase_count(self, counter: Dict[str, int], id: str):
+        if id in counter:
+            counter[id] = counter[id] + 1
+        else:
+            counter[id] = 1
+
     def parse_generator_type_envelope(self, envelopes: types.GeneratorType, parse_objects_func, limit: int = -1):
         indicators = []
         relationships_lst = []
+        parsed_objects_counter: Dict[str, int] = {}
```

Comment:
Please add a comment about this being used for logging purposes

Reply:
[Done](https://github.com/demisto/content/pull/33061/commits/39693e56b65bbdb500e99662e87d81026a11cf43).